examples/b2c/checkout/integrations/classes/B2CDeliverySample.cls

```diff
@@ -247,49 +247,12 @@ global class B2CDeliverySample implements sfdc_checkout.CartShippingCharges {
     }
 
     private List<Id> getOrderDeliveryMethods(List<ShippingOptionsAndRatesFromExternalService> shippingOptions) {
-        String defaultDeliveryMethodName = 'Order Delivery Method';
```

Comment:
@KalyanBoya seems like we have same class in B2B: https://github.com/forcedotcom/commerce-on-lightning/blob/develop/examples/b2b/checkout/integrations/classes/B2BDeliverySample.cls